XUtils.c

```
@@ -259,6 +260,300 @@ size_t strnlen(const char* str, size_t maxLen) {
+static void String_encodeWChar(WCharEncoderState* ps, int c) {
+   assert(!ps->buf || ps->pos < ps->size);
+
+   char* buf = ps->buf;
+   if (buf)
+      buf[ps->pos] = (char)c;
+
+   ps->pos += 1;
+}
+#endif
+
+void EncodePrintableString(WCharEncoderState* ps, const char* src, size_t maxLen, EncodeWChar encodeWChar) {
```

Cppcheck (2.20.0)

[style] The function 'EncodePrintableString' should have static linkage since it is not used outside of its translation unit.

(staticFunction)